content/en/platform/corda/5.0-dev-preview-2/developing/api/api-application.md

```diff
@@ -18,3 +18,35 @@ The `corda-application` module provides the fundamental building blocks required
 - `corda-serialization`.
 
 By depending on `corda-application`, your Cordapp does not need to directly depend on the modules listed above.
+
+`corda-application` provides a number of packages. The key package for defining flows is `flows`, which contains the interfaces to implement and annotations to use to customize flow behaviour. The remaining packages provide some services for use within a flow. A description of each of these packages is provided below. For further details of what is present in each of these packages and what is available via these APIs, consult the API documentation.
```

Comment:
Would suggest using a different word than "key" if possible - it's a bit ambiguous when we're also discussing crypto operations, etc..
main? most important? most significant? crucial? central?